libkbfs/md_id_journal.go

```diff
@@ -29,7 +29,8 @@ type mdIDJournal struct {
 // depend on the ID, as the mdJournal may change the ID when
 // converting to a branch.
 type mdIDJournalEntry struct {
-	ID MdID
+	ID            MdID
+	IsLocalSquash bool
```

Comment:
Is there any pattern to the entries with IsLocalSquash set to true? From the journal_md_ops.go code, it sounds like F*T*, or perhaps F*T*F*? (F* = any number of entries with IsLocalSquash set to false, T* = ... set to true)